### DISCUSSION TITLE
how expose services with ingress-controler/metallb/on-premise ? (like for cicd)

### DISCUSSION THREAD

**Discussion by survivant** [coredns/coredns#4703] [Q&A]
here my usecase. I want to deploy my applications on multiple namespaces

dev
qa
PR-111 (for Pull request)
my services could look like that

chuck-service:8080
quote-service:8080

I'm on premise with nginx-ingress and with metallb as loadbalancer

I'll expose nginx-controler as daemonset with a external IP : 10.1.10.123

for ingress

/chuck -> chuck-service:8080
/quote -> quote-service:8080

I want to have those applications to be accessible for from outside on 10.1.10.123 (I can't expose new IP)

the domain (inside-my-company.com) name it not register in external DNS

ex :
dev.inside-my-company.com/chuck
dev.inside-my-company.com/quote

qa.inside-my-company.com/chuck
qa.inside-my-company.com/quote

....

**Comment by miekg** [comment]
Please ask this on some k8s forum.

thanks.

/label kubernetes
/close
